server.ts

```diff
@@ -190,7 +227,7 @@ async function main() {
     log.info('Finished adding Lockdrop statistics into the DB');
     process.exit(0);
   } else {
```

Comment:
These one-off commands (should_send emails, etc.) don't rely on the app having routes, middleware, api, or error handlers, so now that we have the { ... return; } pattern up above that we're using for the listener, we should move them up so they run faster and the code is easier to read.